app/models/course.rb

```diff
@@ -154,7 +154,7 @@ def self.entry_requirement_options_without_nil_choice
   validate :validate_course_syncable, on: :sync
 
   validate :validate_qualification, :validate_start_date, on: :update
-  validate :validate_qualification, on: :update
+  validate :validate_program_type, on: :update
```

Comment:
just styling

can we collapse 

`validate :validate_field_1, :validate_field_2, :validate_field_3, on: update`

 all in to one

Reply:
Have changed it. Thanks